lib/spack/spack/compilers/nag.py

```diff
@@ -20,6 +21,16 @@ class Nag(Compiler):
                    'f77' : 'nag/nagfor',
                    'fc'  : 'nag/nagfor' }
 
+    @property
+    def openmp_flag(self):
+        return "-openmp"
+
+    @property
+    def cxx11_flag(self):
+        tty.die("cxx11_flag() is not implemented for nag. Consider creating a pull-request.")
+        return "-std=c++11"
```

Comment:
NAG doesn't have a C++ compiler. I'm not sure if you should return no string, die, or just leave out the method since it can't apply to this compiler. But since NAG will usually be combined with another compiler, it may have to return the string usually accepted by most compilers.